Sources/URLSessionDecodable/Parameters.swift

```diff
@@ -48,14 +52,18 @@ public struct JSONParametersEncoder: ParametersEncoding {
 public struct URLParametersEncoder: ParametersEncoding {
```

Comment:
`URLParametersEncoder` doesn't have to rely on `Dictionary`. It could just as well accept `any Collection`

```
public struct URLParametersEncoder<Parameters>: ParametersEncoding where Parameters: Collection<(key: String, value: String)> {
public let parameters: any Parameters
```

In order to have sorting you can now pass in `OrderedDictionary` or `Array<(key: String, value: String)>` or whatever you like. So the library does not have to care about this and no new versions are required to support any other order

Reply:
That's a good idea. Though in that case the responsibility for sorting is transfered to a client code, I think you are right, it is better that way for keeping this library simple.